document_en/appendix.md

```diff
@@ -33,6 +33,8 @@ Copy `atcoder` folder into it, i.e., put it so that the path will be as follows.
 
 - `C:\Program Files (x86)\Microsoft Visual Studio\2019\(Community, Professional or Enterprise)\VC\Tools\MSVC\(Some number, e.g. 14.27.29110)\include\atcoder\dsu.hpp`
 
+Visual Studio only supports 64bit projects. For 32bit(x86) projects, the error *'_umul128': identifier not found* occurs.
```

Comment:
We prefer "the 32bit project don't work, so please change the project setting into 64bit by `Preference > XXX > YYY > ZZZ...`" rather than "32bit doesn't work."